### PR TITLE
Fix java formatting

### PR DESCRIPTION
**Pull request by cormacq**

### CODE REVIEW

File: api/src/main/java/uk/gov/dvsa/mot/persist/TradeReadDao.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package uk.gov.dvsa.mot.persist;
+
+import uk.gov.dvsa.mot.trade.api.Vehicle;
+
+import java.util.Date;
+import java.util.List;
+
+public interface TradeReadDao {
+    //  List<DisplayMotTestItem> getMotHistoryByDateRange( Date startDate, Date endDate ) ;
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/TradeReadDao.java]: Can we get rid of these commented-out code blocks?

File: api/src/main/java/uk/gov/dvsa/mot/persist/jdbc/MotTestReadDaoJdbc.java [hunk 2]
```diff
@@ -0,0 +1,1843 @@
...
+            motTestCurrent.setSite(getSiteById(resultSet.getInt(7)));
+            motTestCurrent.setHasRegistration(resultSet.getBoolean(8));
+            motTestCurrent.setMotTestType(getMotTestTypeById(resultSet.getInt(9)));
+            motTestCurrent.setStartedDate(resultSet.getDate(10));
+            motTestCurrent.setCompletedDate(resultSet.getDate(11));
+            motTestCurrent.setSubmittedDate(resultSet.getDate(12));
+            motTestCurrent.setMotTestStatus(getMotTestStatusById(resultSet.getInt(13)));
+            motTestCurrent.setIssuedDate(resultSet.getDate(14));
+            motTestCurrent.setExpiryDate(resultSet.getDate(15));
+            // TODO causing recursive gets because of stupid dual linking
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/jdbc/MotTestReadDaoJdbc.java:1192]: Stupid to use _stupid_ in a comment .

File: api/src/main/java/uk/gov/dvsa/mot/persist/TradeReadDao.java (hunk 1 above)
Comment (cormacq) [reply api/src/main/java/uk/gov/dvsa/mot/persist/TradeReadDao.java]: Very good question. It's something we didn't look at doing this time round. Can speak with Matthew about this.

File: api/src/main/java/uk/gov/dvsa/mot/persist/jdbc/MotTestReadDaoJdbc.java [hunk 3]
```diff
@@ -0,0 +1,1843 @@
...
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/jdbc/MotTestReadDaoJdbc.java:1244]: Another one.

File: api/src/main/java/uk/gov/dvsa/mot/persist/jdbc/MotTestReadDaoJdbc.java (hunk 2 above)
Comment (cormacq) [reply api/src/main/java/uk/gov/dvsa/mot/persist/jdbc/MotTestReadDaoJdbc.java:1192]: Yeah, I agree. Not a good idea to have it on a public repo either. We did talked about these comments. This wasn't in the scope of the code style changes.

File: api/src/main/java/uk/gov/dvsa/mot/persist/jdbc/MotTestReadSql.java [hunk 4]
```diff
@@ -0,0 +1,675 @@
+package uk.gov.dvsa.mot.persist.jdbc;
+
+/**
+ * SQL queries for the MotTestReadDaoJdbc class to use
+ */
+@SuppressWarnings({"checkstyle:OperatorWrap", "checkstyle:Indentation"})
+class MotTestReadSql {
+    //Horrible optimised query to get the mot test history records
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/jdbc/MotTestReadSql.java]: Horrible adjective.

File: api/src/main/java/uk/gov/dvsa/mot/persist/model/BodyType.java [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+        return this.version;
+    }
+
+    public void setVersion(int version) {
+
+        this.version = version;
+    }
+
+  /*
+   * public List<ModelDetail> getModelDetails() { return this.modelDetails; }
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/model/BodyType.java]: Commented out code.

File: api/src/main/java/uk/gov/dvsa/mot/persist/model/Make.java [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+        return this.version;
+    }
+
+    public void setVersion(int version) {
+
+        this.version = version;
+    }
+
+  /*
+   * public List<Model> getModels() { return this.models; }
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/model/Make.java]: Commented out code.

File: api/src/main/java/uk/gov/dvsa/mot/persist/model/Model.java [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+        return this.make;
+    }
+
+    public void setMake(Make make) {
+
+        this.make = make;
+    }
+
+  /*
+   * public List<ModelDetail> getModelDetails() { return this.modelDetails; }
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/model/Model.java]: Commented out code.

File: api/src/main/java/uk/gov/dvsa/mot/persist/model/MotTestHistory.java [hunk 8]
```diff
@@ -0,0 +1,410 @@
...
+
+        return this.motTestComplaintRef;
+    }
+
+    public void setMotTestComplaintRef(MotTestComplaintRef motTestComplaintRef) {
+
+        this.motTestComplaintRef = motTestComplaintRef;
+    }
+
+    /*
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/model/MotTestHistory.java:257]: Commented out code.

File: api/src/main/java/uk/gov/dvsa/mot/persist/model/MotTestReasonForCancelLookup.java [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+    private boolean isAbandoned;
+    private boolean isDisplayable;
+    private boolean isSystemGenerated;
+    private int createdBy;
+    private Date createdOn;
+    private int lastUpdatedBy;
+    private Date lastUpdatedOn;
+    private int version;
+
+    public MotTestReasonForCancelLookup() {
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/model/MotTestReasonForCancelLookup.java]: Empty method body.

Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/model/MotTestReasonForCancelLookup.java:26]: Empty constructor.

File: api/src/main/java/uk/gov/dvsa/mot/persist/model/MotTestRfrLocationType.java [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+    private String longitudinal;
+    private String vertical;
+    private String sha1ConcatWsChksum;
+    private int createdBy;
+    private Timestamp createdOn;
+    private int lastUpdatedBy;
+    private Timestamp lastUpdatedOn;
+    private int version;
+
+    public MotTestRfrLocationType() {
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/model/MotTestRfrLocationType.java:23]: Empty constructor.

File: api/src/main/java/uk/gov/dvsa/mot/persist/model/MotTestRfrMap.java [hunk 11]
```diff
@@ -0,0 +1,181 @@
...
+    private boolean failureDangerous;
+    private boolean generated;
+    private boolean onOriginalTest;
+    private int createdBy;
+    private Timestamp createdOn;
+    private int lastUpdatedBy;
+    private Timestamp lastUpdatedOn;
+    private int version;
+
+    public MotTestRfrMap() {
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/model/MotTestRfrMap.java:28]: Empty constructor.

File: api/src/main/java/uk/gov/dvsa/mot/persist/model/MotTestRfrMapComment.java [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+    private String comment;
+    private int createdBy;
+    private Timestamp createdOn;
+    private int lastUpdatedBy;
+    private Timestamp lastUpdatedOn;
+    private int version;
+    private MotTestRfrMap motTestCurrentRfrMap;
+    private MotTestHistoryRfrMap motTestHistoryRfrMap;
+
+    public MotTestRfrMapComment() {
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/model/MotTestRfrMapComment.java:22]: Empty constructor.

File: api/src/main/java/uk/gov/dvsa/mot/persist/model/MotTestRfrMapCustomDescription.java [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+    private String customDescription;
+    private int createdBy;
+    private Timestamp createdOn;
+    private int lastUpdatedBy;
+    private Timestamp lastUpdatedOn;
+    private int version;
+    private MotTestRfrMap motTestCurrentRfrMap;
+    private MotTestHistoryRfrMap motTestHistoryRfrMap;
+
+    public MotTestRfrMapCustomDescription() {
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/model/MotTestRfrMapCustomDescription.java:23]: Empty constructor.

File: api/src/main/java/uk/gov/dvsa/mot/persist/model/Organisation.java [hunk 14]
```diff
@@ -0,0 +1,557 @@
...
+    private int slotsBalance;
+    private int slotsOverdraft;
+    private int slotsPurchased;
+    private int slotsWarning;
+    private String tradingName;
+    private Date transitionProcessedOn;
+    private Date transitionScheduledOn;
+    private String vatRegistrationNumber;
+    private int version;
+    // private List<AuthForAe> authForAes;
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/model/Organisation.java:41]: Commented out code.

File: api/src/main/java/uk/gov/dvsa/mot/persist/model/Organisation.java [hunk 15]
```diff
@@ -0,0 +1,557 @@
...
+    // private List<DirectDebit> directDebits;
+    // private List<EventOrganisationMap> eventOrganisationMaps;
+    private CompanyType companyType;
+    private OrganisationType organisationType;
+    // private TransitionStatus transitionStatus;
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/model/Organisation.java:46]: Commented out code.

File: api/src/main/java/uk/gov/dvsa/mot/persist/model/Organisation.java [hunk 16]
```diff
@@ -0,0 +1,557 @@
...
+
+    public Site removeSite(Site site) {
+
+        getSites().remove(site);
+        site.setOrganisation(null);
+
+        return site;
+    }
+
+  /*
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/model/Organisation.java]: Commented out code.

File: api/src/main/java/uk/gov/dvsa/mot/persist/model/TransmissionType.java [hunk 17]
```diff
@@ -0,0 +1,136 @@
...
+        return this.version;
+    }
+
+    public void setVersion(int version) {
+
+        this.version = version;
+    }
+
+  /*
+   * public List<ModelDetail> getModelDetails() { return this.modelDetails; }
```
Comment (raffcecco) [inline api/src/main/java/uk/gov/dvsa/mot/persist/model/TransmissionType.java]: Commented out code.

**Review by raffcecco — APPROVED** [review]
Code style changes look good. FYI, I commented here and there about a few issues relating to inappropriate comments and large commented-out code blocks.

**Pull request closed without merging**